### PR TITLE
Added my essay

### PR DESCRIPTION
**Pull request by jeffreyshen19**

### CODE REVIEW

**Comment by milesmcc** [comment]
Love the essay! And for publishing it open source.

Also—on L40, `target` is set to `_blannk` — perhaps this is supposed to be `_blank`?

_Update: is fixed._

File: essays.html [hunk 1]
```diff
@@ -34,6 +34,15 @@ <h1><a href='index.html'>CSC630: The Open Source Movement</a> // Assignments <sp
         </p>
       </article>
     </section>
+    <section>
+      <article>
+        <p>
+          <a href="./assets/essays/jshen-essay.pdf" target = "_blannk">The Economic Benefits of Open Source Software</a>
```
Comment (milesmcc) [inline essays.html]: Perhaps this is supposed to be `_blank`?

**Comment by milesmcc** [comment]
That was fast! (I'll now never accidentally lose the current tab when clicking on your essay's link.)

**Comment by nzufelt** [comment]
Here come the essays

**Comment by ruddfawcett** [comment]
Hey! Closing this for now due to the sheer quantity of open PRs and the potential for many, many merge conflicts. Please reopen a new pull request once the contributing guidelines have been updated!

**Pull request closed without merging**